system/jlib/jstats.cpp

```diff
@@ -1865,7 +1858,7 @@ void verifyStatisticFunctions()
     //Check the various functions return values for all possible values.
     for (unsigned i1=SMeasureAll; i1 < SMeasureMax; i1++)
     {
-        const char * prefix = queryMeasurePrefix((StatisticMeasure)i1);
+        const char * prefix __attribute__((unused)) = queryMeasurePrefix((StatisticMeasure)i1);
```

Comment:
It is there to check that the function doesn't trigger and internal error.  It could just call the function and have a comment before it.